Lib/ensurepip/__init__.py

```diff
@@ -120,27 +120,27 @@ def _disable_pip_configuration_settings():
     os.environ['PIP_CONFIG_FILE'] = os.devnull
 
 
-def bootstrap(*, root=None, upgrade=False, user=False,
+def bootstrap(*, root=None, prefix=None, upgrade=False, user=False,
```

Comment:
Add `prefix` at the end of the args list, as noted above. Also, supplying *both* root and prefix arguments doesn't really make sense (and may even be rejected by pip at some point in the future, if it isn't already) so it would be better to make these arguments mutually exclusive.

Reply:
Thinking about it, I'm not sure they should be mutual exclusive.

If `root` equals `$(DESTDIR)` and `prefix` equals `$(prefix)`, I would guess we want to install into `$(DESTDIR)/$(prefix)`. I need to look more closely at this later today.